vhd/lib/libvhd.c

```diff
@@ -2726,7 +2727,7 @@ vhd_open(vhd_context_t *ctx, const char *file, int flags)
 		dev_t mapper_dev;
 		if (drbd_is_up_to_date(&stats) && drbd_to_mapper(&stats, &mapper_dev) == 0) {
 			if (snprintf(bypass_file, sizeof bypass_file, "/dev/block/%u:%u", major(mapper_dev), minor(mapper_dev)) == -1)
```

Comment:
Not really related to this PR, but should it also check for `snprintf() >= sizeof(bypass_file)`?

Reply:
Quite probably but I'll let XenOrchestra worry about that.

Reply:
`bypass_file` is big enough to contain `snprintf(...)` characters in this case :wink:: https://github.com/xapi-project/blktap/blob/74ecccfb33a59bc1ca8ade9a8f85070ed7bda0e8/vhd/lib/libvhd.c#L2715